src/clr-angular/data/datagrid/datagrid-column-separator.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Component, Renderer2 } from '@angular/core';
+
+import { ColumnResizerService } from './providers/column-resizer.service';
+import { TableSizeService } from './providers/table-size.service';
+
+let nbCount: number = 0;
+
+@Component({
+  selector: 'clr-dg-column-separator',
+  template: `
+    <button class="datagrid-column-handle" tabindex="-1" type="button"
+      clrDraggable 
+      [clrGroup]="columnSeparatorId" 
+      (clrDragStart)="columnResizerService.startResize(); showTracker(resizeTrackerEl)" 
```

Comment:
`showTracker()` should handle the call to the service. Having more than one operation inline in an HTML output is generally a red flag.